modules/pwa/index.ts

```diff
@@ -23,6 +23,9 @@ export default defineNuxtModule<VitePWANuxtOptions>({
     }
     let webmanifests: LocalizedWebManifest | undefined
 
+    nuxt.options.appConfig = nuxt.options.appConfig || {}
+    nuxt.options.appConfig.pwaEnabled = !nuxt.options.dev || process.env.VITE_DEV_PWA === 'true'
```

Comment:
how about tauri, rn it is disabled

Reply:
This PR doesn't change the current behaviour, just moves the setting from `nuxt.config` -> `pwa/index`. Do you think we need to change the current behaviour - or am I missing something?

Reply:
`nuxt.options.appConfig.pwaEnabled = !nuxt.options.dev || process.env.VITE_DEV_PWA === 'true'` that's true also on Tauri

Reply:
I mean, `!nuxt.options.dev`

Reply:
Current login is the same: https://github.com/elk-zone/elk/blob/main/nuxt.config.ts#L101